src/modules/did/did.service.ts

```diff
@@ -215,7 +225,12 @@ export class DIDService implements OnModuleInit, OnModuleDestroy {
         logs: JSON.stringify(logs),
       });
 
-      return this.didRepository.save(updatedEntity);
+      const updated = await this.didRepository.save(updatedEntity);
+      await this.didSyncStatusRepository.save({
+        document: did,
+        status: DidSyncStatus.Synced,
+      });
```

Comment:
I think this should be in a transaction as I think an invariant should be that there is always a `didSyncStatus` entity if there is a `did` entity.